HTML/EN/plugins/Qobuz/settings/basic.html

```diff
@@ -40,6 +40,25 @@
 				<option [% prefs.showYearWithAlbum ? "" : "selected" %] value="0" >[% "NO" | string %]</option>
 			</select>
 		[% END %]
+		<hr style="opacity:0.4" class="sub-sep"/>
```

Comment:
I personally try to avoid hard-coded formatting in the templates. This should be the skin's duty to get right.

Reply:
Done.